src/components/sectionList/listView/useModelListView.tsx

```diff
@@ -148,7 +148,9 @@ export const useModelListView = () => {
     const selectedView = query.data || defaultView
 
     const columns = selectedView.columns
-    const filters = selectedView.filters
+    const filters = query.data?.filters?.length
+        ? query.data.filters
+        : defaultView.filters
```

Comment:
Why is this necessary? In a recent PR I made it possible to remove all filters, and we shouldn't render default if a user deliberately deselects all filters?

Reply:
So `query.data.filters` shows an empty array in categoryOptionGroup while `defaultView.filters` contains the the "dymensionType" filter in the array, hence the logic added.

Reply:
But you would just have to select it in `ManageView`? This just means that user has done changes to the view?